test/e2e/e2e_test.go

```diff
@@ -43,6 +44,16 @@ import (
 
 var _ = ginkgo.Describe("Kueue visibility server", func() {
 	const defaultFlavor = "default-flavor"
+
+	// We do not check Name and OwnerReference's UID as they are generated
+	// We do not check Controller and BlockOwnerDeletion as we intentionally leave it empty
+	var pendingWorkloadsCmpOpts = []cmp.Option{
+		cmpopts.IgnoreFields(metav1.ObjectMeta{}, "Name"),
+		cmpopts.IgnoreFields(metav1.OwnerReference{}, "UID"),
+		cmpopts.IgnoreFields(metav1.OwnerReference{}, "Controller"),
```

Comment:
Is it needed in the ignored fields since we set it to nil ? I guess it makes some value to test it is nil